biocypher_metta/adapters/gene_ontology_adapter.py

```diff
@@ -0,0 +1,116 @@
+import rdflib
+from owlready2 import *
+from biocypher_metta.adapters.ontologies_adapter import OntologyAdapter
+
+class GeneOntologyAdapter(OntologyAdapter):
+    def __init__(self, write_properties, add_provenance, type, label='ontology term', dry_run=False):
+        super().__init__(write_properties, add_provenance, label, type, dry_run)
+        self.source = "Gene Ontology"
+        self.source_url = "http://purl.obolibrary.org/obo/go.owl"
+        self.ONTOLOGIES = {
+            'go': 'http://purl.obolibrary.org/obo/go.owl'
+        }
+
+
+    def get_graph(self, ontology='go'):
+        if ontology not in self.ONTOLOGIES:
+            raise ValueError(f"Ontology '{ontology}' is not defined in this adapter.")
+        
+        onto = get_ontology(self.ONTOLOGIES[ontology]).load()
+        self.graph = default_world.as_rdflib_graph()
+        self.clear_cache()
+        return self.graph
+
+    def get_nodes(self):
```

Comment:
What's the difference between the `get_nodes` and `get_edges` methods in this class and in the `OntologyAdapter` parent class? I don't see much of a difference. If there is no difference, perhaps no need to override it here.